src/Microsoft.AspNetCore.Mvc.TagHelpers/Internal/FileVersionProvider.cs

```diff
@@ -111,7 +111,14 @@ public string AddFileVersionToPath(string path)
 
         private static string GetHashForFile(IFileInfo fileInfo)
         {
-            using (var sha256 = SHA256.Create())
+            HashAlgorithm sha256;
+#if NETSTANDARD1_6
+            sha256 = SHA256.Create();
+#else
+            sha256 = new SHA256Cng();
```

Comment:
I think this has to be `SHA256CryptoHashProvider` - https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Common/CryptoHashProvider.cs#L107